Classes/Command/ExtensionCommandController.php

```diff
@@ -62,14 +63,16 @@ class ExtensionCommandController extends CommandController
      * Activates one or more extensions by key.
      * Marks extensions as active, sets them up and clears caches for every activated extension.
      *
-     * This command is only available in non composer mode.
+     * This command is deprecated (and hidden) Composer mode.
```

Comment:
Hehe, you forgot the important part: the _in_. ;-)

Reply:
done now